build-logic/documentation/src/main/groovy/gradlebuild/docs/GradleJavadocsPlugin.java

```diff
@@ -74,16 +74,26 @@ private void generateJavadocs(Project project, ProjectLayout layout, TaskContain
 
             // TODO: This should be part of Javadoc task
             task.getInputs().file(javadocs.getJavadocCss())
-                    .withPropertyName("stylesheetFile")
-                    .withPathSensitivity(PathSensitivity.NAME_ONLY);
+                .withPropertyName("stylesheetFile")
+                .withPathSensitivity(PathSensitivity.NAME_ONLY);
 
             StandardJavadocDocletOptions options = (StandardJavadocDocletOptions) task.getOptions();
-            options.setEncoding("utf-8");
-            options.setDocEncoding("utf-8");
-            options.setCharSet("utf-8");
+            //options.setEncoding("utf-8");
```

Comment:
❓ Is this really not needed?

If yes, then let's fully delete the code.

Reply:
I updated Javadocs to HTML5, so it's no longer needed. Deleting permanently.

Reply:
Undoing this.